tools/jenkins/run_distribution.sh

```diff
@@ -134,12 +124,11 @@ elif [ "$platform" == "macos" ]; then
         ;;
     esac
 
-    # clean up
-    cd ~/ 
+    # Clean up
     rm -rf $brew_root
 
     # Make sure the system brew installation is still unaffected
-    export PATH=$OLD_PATH
+    export PATH=$ORIGINAL_PATH
```

Comment:
when you set an env var in bash, the variable is local to the current process and its child processes. So whatever you do in this bash, you can never mess with other processes' env vars. Right?